docs/pages/configuration/images/image-definition/tags.mdx

```diff
@@ -16,16 +16,16 @@ images:
     tags:
     # The five # are replaced with 5 random characters. The variable ${devspace.git.branch} is replaced
     # with the current branch the user is on
-    - 'dev-${devspace.git.branch}-#####' 
+    - 'dev-${devspace.git.branch}-#####'
 ```
- 
+
 :::info Unique Tags For Development
 **Make sure tags are unique** when defining a custom tagging schema for development. Unique tags ensure that your application gets started with the newly built image instead of using an older, cached version. Therefore, it is highly recommended for non-production tags to either use `#` placeholders or `devspace.timestamp` as a suffix in your tagging schema (see [example below](#example-custom-tagging-schema)).
```

Comment:
```suggestion
**Make sure tags are unique** when defining a custom tagging schema for development. Unique tags ensure your application starts with the newly built image instead of using an older, cached version. Therefore, it is highly recommended for non-production tags to either use `#` placeholders or `devspace.timestamp` as a suffix in your tagging schema (see [example below](#example-custom-tagging-schema)).
```